src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -553,5 +553,15 @@ public static string TryGetAndroidJarPath (TaskLoggingHelper log, string platfor
 			}
 			return Path.Combine (platformPath, "android.jar");
 		}
+
+		public static Dictionary<string, string> LoadResourceCaseMap (string resourceCaseMap)
+		{
+			var result = new Dictionary<string, string> ();
+			if (resourceCaseMap != null) {
+				foreach (var arr in resourceCaseMap.Split (';').Select (l => l.Split ('|')).Where (a => a.Length == 2))
+					result [arr [1]] = arr [0]; // lowercase -> original
```

Comment:
This is probably for the future, but I wonder if we could use `Span<T>` here to make this faster.

This current code looks like it would allocate a lot of strings.